t/core.t

```diff
@@ -17,7 +17,11 @@ BEGIN {
 initialize_tests();
 
 diag("Create Whim object");
-my $whim = Whim::Core->new( { data_directory => $Whim::Core::TRANSIENT_DB } );
+my $whim = Whim::Core->new(
+    {   data_directory         => $Whim::Core::TRANSIENT_DB,
+        author_photo_directory => "$FindBin::Bin/public/author_photos",
```

Comment:
Should this be `$FindBin::Bin` or `$FindBin::Bin/..`?

Reply:
Correct as written, I do believe. $FindBin::Bin resolves to `t/core.t`, and the test `public/` directory is a sibling to it.